setup.py

```diff
@@ -38,6 +38,8 @@
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
```

Comment:
shall we change the `install_requires` limit to `< 5.0`?

Reply:
Existing `selenium ~= 4.1` limits the version up to 4.x, so it is not necessary